package.json

```diff
@@ -4,38 +4,30 @@
   "private": false,
   "description": "A collection of JavaScript libraries for Guardian projects",
   "homepage": "https://github.com/guardian/libs#readme",
-  "bugs": {
-    "url": "https://github.com/guardian/libs/issues"
-  },
+  "bugs": { "url": "https://github.com/guardian/libs/issues" },
   "repository": "https://github.com/guardian/libs.git",
   "license": "Apache-2.0",
   "author": "Guardian developers",
   "sideEffects": false,
   "main": "dist/cjs/index.js",
-  "unpkg": "dist/umd/index.min.js",
   "module": "dist/esm/index.js",
-  "source": "src/index.ts",
-  "types": "dist/esm/index.d.ts",
-  "files": [
-    "dist",
-    "src"
-  ],
+  "types": "dist/types/index.d.ts",
+  "files": ["dist", "src"],
   "scripts": {
-    "build": "npm-run-all clean --parallel compile:*",
+    "prebuild": "yarn clean",
+    "build": "pkgu build",
     "clean": "rm -rf dist",
-    "compile:common": "tsc --project ./tsconfig.build.json --outDir ./dist/cjs --module CommonJS",
-    "compile:esm": "tsc --project ./tsconfig.build.json --outDir ./dist/esm",
-    "compile:umd": "rollup -c",
```

Comment:
Minor: are we dropping UMD support?

Reply:
i think so? it's not in the recs, do we need it? or alternatively, should we discourage it?